scripts/editor/register-blocks.js

```diff
@@ -268,43 +276,36 @@ export const getSharedAttributes = (globalManifest, blockManifest) => {
 };
 
 /**
- * Iterate over component object in block manifest and search and replace the component attributes with new one.
- * Search and replace the component attributes with new one.
+ * Iterate over attributes or example attributes object in block/component manifest and append the parent prefixes.
  *
- * @param {object} component         - Object of component manifests to iterate.
- * @param {string} realComponentName - React component name defined in the component manifest.
- * @param {string} newComponentName  - New component name to search and replace the original.
- * @param {string} [key=attributes]  - Type of output, can be: `attributes` or `example`.
+ * @param {object} manifest           - Object of component/block manifest to get data from.
+ * @param {boolean} [isExample=false] - Type of items to iterate, if true example key will be use, if true attributes will be used.
```

Comment:
```suggestion
 * @param {boolean} [isExample=false] - Type of items to iterate, if false example key will be use, if true attributes will be used.
```
I think one should be false. It could be the other part of the sentence.

Reply:
Yeah, I added this as a suggestion as well, not sure why it's not picked up 🤷‍♂️